### DISCUSSION TITLE
Idea: extend SetOption30 hassio integration so 2 is Cover, 3 is Fan, 4 is Lock, 5 is Siren

### DISCUSSION THREAD

**Discussion by MarkAtwood** [arendst/Tasmota#16924] [Ideas]
I know this can be done using the hassio "switch_as_x", but I would prefer to be able to do it at the Tasmota level.

**Comment by barbudor** [comment]
It doesn't make much sense as those terms means nothing for Tasmota.
Tasmota announce what it is (a switch) not how it is used
You have friendlynames for that

Beside, there is a little technical issue that SetOption30 is a binary/boolean SetOption which cannot easily be changed to support any other value than 0 or 1
